cwd/editor.js

```diff
@@ -55,54 +112,57 @@ const makeNumInput = (id, div, num, ref) => {
 
 let viewItr = 0;
 const views = ['http://localhost:3006/glyphs/5d3744e584687c30ef9e5b47/', 'http://localhost:3006/glyphs/5d37452784687c30ef9e5b48/',
-'http://localhost:3006/glyphs/5d4885f18227675bc73b98ac/']
-
-views.forEach(view => {$.getJSON(view, function(data) {
-  let viewMsgItr = 0;
+  'http://localhost:3006/glyphs/5d4885f18227675bc73b98ac/'
+]
```

Comment:
This cannot go live until we retrieve these views from the server and make these paths relative.  Note the `API_URL` variable that is provided through the server.

Reply:
This was fixed, as we added the relevant API path.